xonsh/lib/os.xsh

```diff
@@ -0,0 +1,20 @@
+"""Xonsh extension of the standard library os module, using xonsh for
+subprocess calls"""
+import sys
+
+from xonsh.imphooks import install_hook
+
+install_hook
+
+import os
+from contextlib import contextmanager
+
+@contextmanager
+def indir(d):
+    """Context manager for temporarily entering into a directory.
+     Copyright (c) 2017, Anthony Scopatz
+     Copyright (c) 2018, The Regro Developers"""
```

Comment:
I would throw the copyrights into comments, rather than the docstring.  They don't really help the user. Also, you can leave off mine :)